.github/workflows/full-ci.yml

```
@@ -11,7 +11,7 @@ on:
 
 env:
   # Note: Use exactly match version of tool, to avoid unexpected issues with test on CI.
-  GO_VERSION: '1.22.11'
+  GO_VERSION: '1.23.1'
```

It is better to use a latest version.
```suggestion
  GO_VERSION: '1.23.8'
```